docs/release-notes.rst

```diff
@@ -6,6 +6,51 @@
  Release Notes
 ###############
 
+**************
+ Version 0.37
+**************
+
+Version 0.37.0
+==============
+
+**Release Date:** September 26, 2024
+
+**Breaking Changes**
+
+-  API: Remove model_hub library from determined.
+
+   -  Starting with this release, MMDetTrial and BaseTransformerTrial are removed. HuggingFace users
+      should look at provided `HuggingFace TrainerAPI
+      examples<https://github.com/determined-ai/determined/tree/main/examples/hf_trainer_api>_`,
```

Comment:
This link does not work in the preview https://determined-ai-docs.s3.us-west-2.amazonaws.com/previews/d0b11d38eaa8566308800c163206f076/release-notes.html